docs/global-infra-deployment.md

```diff
@@ -10,3 +10,8 @@ make aws-vault
 ```shell
 make deploy-global-infra
 ```
+
+## Deploy global tools
+```shell
+make deploy-global-tools
+```
```

Comment:
I think it would be nice to add a sentence or two about what these tools are and mention that they are optional.